### DISCUSSION TITLE
Position and rotation data of Stretch is not updating after movement

### DISCUSSION THREAD

**Discussion by rajas1310** [empriselab/RCareWorld#120] [Help - Programming] [answered]
Hi organizers,
I observed that the `robot.data["position"] = [0.11,-1.19,1.21]` & `robot.data["rotation"]  = [0,0,0]` irrespective of the actual Stretch position and orientation.

Also the same code generates a slightly different motion of the robot. Is there any random seed parameter used in the unity environment that we can't access?

**Comment by YoruCathy** [comment, marked as answer]
Try robot.data["positions"][0].
Here's an example: https://github.com/empriselab/RCareWorld/blob/6f0b816372c12449f8df2d4ec387722e283fb1b5/tests/test_bathing_env.py#L79

**Reply by rajas1310** [reply]
It worked. Thanks!
